api/nakadi-event-bus-api.yaml

```diff
@@ -1126,12 +1149,21 @@ definitions:
         example: '1996-12-19T16:39:57-08:00'
       received_at:
         type: string
+        readOnly: true
         description: |
           Timestamp of the reception of the Event by Nakadi. This is enriched upon reception of
           the Event.
           If set by the producer Event will be rejected.
         format: date-time
         example: '1996-12-19T16:39:57-08:00'
+      version:
+        type: string
+        readOnly: true
+        description: |
+          Version of the schema used for validating this event. This is enriched upon reception.
+          Clients consuming events should deserialize events only when differences between their
+          schema and the one used to validate differs on minor level, for example, from '1.1.0' to
+          '1.2.0'.
```

Comment:
The meaning of the version is well defined as following semver https://github.com/zalando/nakadi/pull/469/files#diff-4492c6b26ee0a37518728ec1585420efR1641

So users are safe to interpret it. Actually, as by our experience, every field that is human readable will be interpreted.

This description gives a hint on how to implement clients in a defensive mindset, since there is no independent channel to notify about changes, we encourage clients to be aware of changes by inspecting this field.